generators/client/index.js

```diff
@@ -187,19 +200,30 @@ module.exports = JhipsterClientGenerator.extend({
             insight.track('generator', 'app');
             insight.track('app/useSass', this.useSass);
             insight.track('app/enableTranslation', this.enableTranslation);
+            insight.track('app/nativeLanguage', this.nativeLanguage);
+            insight.track('app/languages', this.languages);
         },
 
         configureGlobal: function () {
             // Application name modified, using each technology's conventions
             this.angularAppName = this.getAngularAppName();
-            this.camelizedBaseName = _.camelize(this.baseName);
-            this.slugifiedBaseName = _.slugify(this.baseName);
+            this.camelizedBaseName = _s.camelize(this.baseName);
+            this.slugifiedBaseName = _s.slugify(this.baseName);
             this.lowercaseBaseName = this.baseName.toLowerCase();
+            this.nativeLanguageShortName = this.enableTranslation ? this.nativeLanguage.split("-")[0] : 'en';
         },
 
         saveConfig: function () {
             this.config.set('useSass', this.useSass);
-            this.config.set('enableTranslation', this.enableTranslation);
+            if (this.enableTranslation === true) {
```

Comment:
Yes there is. When running the app generator the server subgenerator is run first which promts for translation questions. Later on when it is time for the client generator to run the enableTranslate is undefined cause of [line 181](https://github.com/jhipster/generator-jhipster/pull/2970/files#diff-0aacb43c4d7b6935003fe4580a092707R188). 
Then the client subgenerator will overwrite the the setting stored in by the server subgenerator. 
I know there is probably a better way to do this but I have not had time to think about this. There have been a hell trying to run a subgenerator from a subgenerator and share the config for some reason. Sharing data amongst the subgenerators is not that obvious to me.